weblate/vcs/base.py

```diff
@@ -50,7 +50,11 @@ def __init__(self, retcode, message):
 
     def get_message(self):
         if self.retcode != 0:
-            return "{0} ({1})".format(self.args[0], self.retcode)
+            fork_url = "https://{0}:{1}@github.com".format(
+                settings.GITHUB_USERNAME, settings.GITHUB_TOKEN
+            )
+            error_message = cleanup_repo_url(fork_url, text=self.args[0])
+            return "{0} ({1})".format(error_message, self.retcode)
```

Comment:
Sorry for confusion - you don't need to introduce calls to `cleanup_repo_url`, just extend it to cleanup GITHUB credentials.